src/main/java/com/minecolonies/coremod/colony/buildings/BuildingHome.java

```diff
@@ -324,21 +311,6 @@ public boolean hasResident(final CitizenData citizen)
         return residents.contains(citizen);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This call to "contains()" may be a performance hot spot if the collection is large. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2250)